packages/bundler-plugin-core/src/sentry/telemetry.ts

```diff
@@ -76,3 +77,32 @@ export function captureMinimalError(error: unknown | Error, hub: Hub) {
 
   hub.captureException(sentryError);
 }
+
+export function addPluginOptionTags(options: InternalOptions, hub: Hub) {
+  const {
+    cleanArtifacts,
+    finalize,
+    setCommits,
+    injectReleasesMap,
+    dryRun,
+    errorHandler,
+    deploy,
+    include,
+  } = options;
+
+  hub.setTags({
+    include: include.length > 1 ? "multiple-entries" : "single-entry",
+    "clean-artifacts": cleanArtifacts,
+    "finalize-release": finalize,
+    "add-deploy": deploy ? true : false,
+    "inject-releases-map": injectReleasesMap,
+    "dry-run": dryRun,
+    "error-handler": errorHandler ? "custom" : "none",
+  });
+
+  if (setCommits) {
+    hub.setTag("set-commits", setCommits.auto === true ? "auto" : "manual");
+  } else {
+    hub.setTag("set-commits", false);
```

Comment:
m: We can just conditionally set this tag, there’s a `has:X` query we can use for an existence check when querying in Sentry.

Reply:
Sounds good to me! [92b6638](https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/113/commits/92b66383c1948754c1760e89ba2af89fd1365708)